packages/client/lib/sync/fullsync.ts

```diff
@@ -8,6 +8,11 @@ import { VMExecution } from './execution/vmexecution'
 import { TxPool } from './txpool'
 import { Event } from '../types'
 
+interface sentBlock {
+  hash: string
+  timeSent: number
```

Comment:
Also here: please remain consistent on naming with the tx pool naming (there: `added`). This can very well mean that you rename the tx pool name depending on what you find more appropriate, we just shouldn't name differently for the same things.

Reply:
I've updated this to `handledObject` to match the tx pool.